tests/scripts/analyze_outcomes.py

```diff
@@ -15,25 +15,31 @@
 
 import check_test_cases
 
-class Results:
+class TestLog:
```

Comment:
The job of this class is to keep track of errors, not just to log them. So `Results` is a good name and `TestLog` is misleading.